### ISSUE TITLE
Auto Connect to Voice Channel

### ISSUE THREAD

**Issue by n8acl** [jess-sys/dmr-bridge-discord#4]
HI! Your favorite pest. 

This is a suggestion not an issue, but just wanted to put this for tracking. 

Would it be possible to give the bot a specific voice channel to connect to on startup? Leave the join command to let users move it around if needed, but on startup it would be awesome to have it auto connect to a default channel. I know it can be done in Python, but since I don't know Rust (I really need to sit down and devote time to learning it.) I don't know how hard that would be to do. I do it for some other bots that I have written in Python for voice clients so I know how to do it there. 

I have a couple of the bots running in two servers and it is a pain to have to connect to each voice channel and issue the join command when I loose internet here at night (thanks spectrum or pushing updates that take me down sometimes). 

Thanks! Appreciate the consideration :)

**Comment by sprshr** [comment]
I'm not too fond of the idea of having the bot remain in a voice chat 24/7, in which case Discord will keep showing a speaker icon on top of your server icon all the time. However, restricting the bot to a single voice channel would be quite helpful especially when it comes to moderation and ensuring only users with a certain role (that would be those who are licensed) can talk/transmit.

**Comment by n8acl** [comment]
Exactly. That is the exact reason I do it. :) I actually have multiple bots in multiple voice channels in my private server so I do have the speaker icon all the time, but that doesn't bother me all that much. :) Thanks for at least looking at it :)
